### DISCUSSION TITLE
What do we need to build to learn most about our problem

### DISCUSSION THREAD

**Discussion by ks0m1c** [ve1ld/vyasa#4] [Scope]
Where we plot ahead steps that  
1) would reveal the most about the problem we are working on 
2) and the order of those steps in which to proceed

P.S take good care in being perceptive to that which it unveils and what its the relationship to the next plot at the end of proceess

**Comment by rtshkmr** [comment]
## Problem Definition 

### Some thoughts on Problem Defn: 
1. individual journey
2. fragmented info
3. sharing has issues if it doesn't promote new viewers to explore the content further  
   * sharing can have various uses: sharing just to share the content (effective if it's limited places for resting within the website) 
   * who is more impt (sharer vs sharee) 
   * how important is the social sharing? do they just want a way to share tgt for private discussions (to be determined later in the dev via tests)


## Random Thoughts:
1. approach can be READ-heavy or WRITE-heavy? 


## Some requirements:

1. Data Ingestion Engine : (READ) 
   * what resources to use and to scrape it?
2.

**Reply by rtshkmr** [reply]
Comparing the 3 problems: 
- fragmented not as big of a problem because we can work with one discrete text / content for now, and side-step this issue
- sharing is likely more worthwhile because we can explore people's actual expected user journeys 
- indiv journey --> probably contains the actual value-add that we'll be doing when compared to other such projs

**Reply by rtshkmr** [reply]
It's like the lurker, poster split -- readers will be the majority, rather than the writer
- power law relationship == 80/20 split 
- this might make us prioritise READ features more

**Reply by rtshkmr** [reply]
## Conclusions: 
1. Read focused 
2. Good-morning-as-a-service

**Comment by rtshkmr** [comment]
## Questions 

1. How to do clustering by literary themes? 
   - socially generated tagging by user-interactions 
   - Similar to [community notes feature](https://communitynotes.twitter.com/guide/en/under-the-hood/ranking-notes) on twitter ==> **_interpreting reality together_**
   - this is pegged to us getting the necessary volume for this
